db/schema.rb

```diff
@@ -141,6 +141,13 @@
     t.index ["nct_id"], name: "index_wiki_pages_on_nct_id", unique: true
   end
 
+  create_table "word_frequencies", force: :cascade do |t|
```

Comment:
Does this need to be here? Is that the table for type ahead?

Reply:
I didn't put that there... Seems like it was from a merge that Nick did

Reply:
I'm pretty sure it doesn't belong here.  Once you take it out I'll merge this PR.

Reply:
I removed it.